lib/common/email.sh

```diff
@@ -165,7 +166,11 @@ get_uploader_email() {
             return 0
         fi
     fi
-    log_error "Could not find uploader for file '$file'"
+
+    local error_msg = "Could not find uploader for file '$file'"
+    log_error "'$error_msg'"
```

Comment:
Just wondering if we need to log twice here, given that we provide the error message below too?